src/reformat.c

```diff
@@ -194,33 +194,32 @@ static avifBool avifPrepareReformatState(const avifImage * image, const avifRGBI
 }
 
 // Formulas 20-31 from https://www.itu.int/rec/T-REC-H.273-201612-I/en
-static int avifReformatStateYToUNorm(avifReformatState * state, float v)
+static int avifReformatStateYToUNorm(avifYUVColorSpaceInfo * info, float v)
 {
-    int unorm = (int)avifRoundf(v * state->yuv.rangeY + state->yuv.biasY);
-    return AVIF_CLAMP(unorm, 0, state->yuv.maxChannel);
+    int unorm = (int)avifRoundf(v * info->rangeY + info->biasY);
+    return AVIF_CLAMP(unorm, 0, info->maxChannel);
 }
 
-static int avifReformatStateUVToUNorm(avifReformatState * state, float v)
+static int avifReformatStateUVToUNorm(avifYUVColorSpaceInfo * info, float v)
```

Comment:
Nit: These two functions, `avifReformatStateYToUNorm` and `avifReformatStateUVToUNorm`, have objected-oriented style names -- the function names start with the type of the first parameter. To preserve this naming convention (commonly used in libavif), we would need to rename them `avifYUVColorSpaceInfoYToUNorm` and `avifYUVColorSpaceInfoUVToUNorm`. But those names become quite long.